api/src/integrations/epic/inspections-utils.js

```diff
@@ -41,15 +42,17 @@ class Inspections {
     // Apply common Epic pre-processing/transformations
     epicRecord = EpicUtils.preTransformRecord(epicRecord);
 
-    // Generate a link that will get us the document when placed in an href.
-    const attachments = [];
+    // Creating and saving a document object if we are given a link to an EPIC document.
+    const documents = [];
     if (epicRecord._id && epicRecord.documentFileName) {
-      attachments.push({
-        url: `https://projects.eao.gov.bc.ca/api/document/${epicRecord._id}/fetch/${encodeURIComponent(
+      const savedDocument = await DocumentController.createLinkDocument(
+        epicRecord.documentFileName,
+        (this.auth_payload && this.auth_payload.displayName) || '',
+        `https://projects.eao.gov.bc.ca/api/document/${epicRecord._id}/fetch/${encodeURIComponent(
           epicRecord.documentFileName
-        )}`,
-        fileName: epicRecord.documentFileName
-      });
+        )}`
+      );
+      documents.push(savedDocument);
     }
```

Comment:
It would be good to pull this out into epic-utils.js, since it will get used in all of the epic imports.

Reply:
it is pulled out. It's using a function in the DocumentController